src/protocol/ppmout.c

```diff
@@ -66,7 +68,7 @@ static void build_data_pkt()
 static u16 ppmout_cb()
 {
     build_data_pkt();
-    PPM_Enable(Model.proto_opts[NOTCH_PW], pulses, Model.num_channels+1);
+    PPM_Enable(Model.proto_opts[NOTCH_PW], pulses, Model.num_channels+1, Model.proto_opts[POLARITY]);
```

Comment:
we need handle proto_opts change.

Reply:
That's not worth it as it's set every time the callback is called.